tsdb/head.go

```diff
@@ -750,6 +760,10 @@ func (h *Head) Init(minValidTime int64) error {
 		return errors.Wrap(err, "finding WAL segments")
 	}
 
+	WALReplayStatus.Started = true
```

Comment:
I think there is a race here. What if API is reading from the variable in the moment it's writing? Unfortunately we might want to add some concurrency control (mutexes)